CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+0.4.6 (2023-09-21)
+------------------
+
+* Increased the client timeout to match the server’s.
```

Comment:
Is the exact timeout value finalized? Could you please double-check the latest conversation?